solr/CHANGES.txt

```diff
@@ -119,7 +119,8 @@ Deprecation Removals
 
 Dependency Upgrades
 ---------------------
-(No changes)
+
+* SOLR-16984: Upgraded internal Jetty server from Jetty 10 to Jetty 12, including migration from javax.* to jakarta.* namespaces for Servlet and related APIs. (Sanjay Dutt, David Smiley)
```

Comment:
I'm not fully convinced other dependencies need to be upgraded, but if this needs to upgrade other dependencies (spring, aws, etc) they should probably be called out as well